grid-cli/internal/cmd/get.go

```diff
@@ -8,6 +8,16 @@ import (
 	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
 )
 
+func SetOrAppend(t deployer.TFPluginClient, contractIDs map[uint32]uint64, node uint32, contractID uint64) {
+	if len(contractIDs) == 0 {
+		t.State.CurrentNodeDeployments[node] = []uint64{contractID}
+
+	} else {
+		t.State.CurrentNodeDeployments[node] = append(t.State.CurrentNodeDeployments[node], contractID)
```

Comment:
make sure contractId is not set inside the list(avoid adding it twice for example)